packages/datadog_flutter_plugin/README.md

```diff
@@ -7,7 +7,7 @@ Datadog RUM SDK versions >= 1.4 support monitoring for Flutter 3.0+.
 
 ## Current Datadog SDK Versions
 
-[//]: # "SDK Table"
+[//]: # (SDK Table)
```

Comment:
Out of curiosity, what's this syntax doing in Markdown?

Reply:
It's essentially a comment. It makes Markdown not show the rest of the line: https://stackoverflow.com/questions/4823468/comments-in-markdown